static/app/views/issueList/addViewPage.tsx

```diff
@@ -0,0 +1,286 @@
+          */}
+          <StyledOverflowEllipsisTextContainer>
+            {suggestion.label}
+          </StyledOverflowEllipsisTextContainer>
+          <QueryWrapper>
+            <FormattedQuery query={suggestion.query} />
+            {
+              <ActionsWrapper>
+                <StyledButton
+                  size="zero"
+                  onClick={e => {
+                    e.stopPropagation();
+                    onNewViewSaved?.(suggestion.label, suggestion.query, true);
+                  }}
+                  borderless
+                >
+                  {t('Save as new view')}
+                </StyledButton>
+              </ActionsWrapper>
+            }
+          </QueryWrapper>
+          <StyledInteractionStateLayer />
+        </Suggestion>
+      ))}
+    </SuggestionList>
+  );
+}
+
+function FeedbackButton() {
+  const openForm = useFeedbackForm();
+
+  if (!openForm) {
+    return null;
+  }
+
+  return (
+    <Button
+      size="xs"
+      icon={<IconMegaphone />}
+      onClick={() =>
+        openForm({
+          messagePlaceholder: t('How can we make custom views better for you?'),
+          tags: {
+            ['feedback.source']: 'custom_views',
+            ['feedback.owner']: 'issues',
+          },
+        })
+      }
+      style={{width: 'fit-content'}}
+    >
+      {t('Give Feedback')}
+    </Button>
+  );
+}
+
+export default AddViewPage;
+
+const SavedSearchesTitle = styled('div')`
+  align-items: center;
+  display: flex;
+  gap: ${space(1)};
+`;
+
+const StyledInteractionStateLayer = styled(InteractionStateLayer)`
+  border-radius: 4px;
+`;
+
+const StyledOverflowEllipsisTextContainer = styled(OverflowEllipsisTextContainer)`
+  margin-left: ${space(1)};
+`;
+
+const TitleWrapper = styled('div')`
+  color: ${p => p.theme.subText};
+  font-weight: 550;
+  font-size: ${p => p.theme.fontSizeMedium};
+  margin-bottom: ${space(0.75)};
+`;
+
+const ActionsWrapper = styled('div')`
+  display: flex;
+  align-items: center;
+  gap: ${space(0.75)};
+  visibility: hidden;
+`;
+
+const StyledButton = styled(Button)`
+  font-size: ${p => p.theme.fontSizeMedium};
+  color: ${p => p.theme.subText};
+  font-weight: ${p => p.theme.fontWeightNormal};
+  padding: ${space(0.5)};
+  border: none;
+  margin-right: ${space(1.5)};
+
+  &:hover {
+    color: ${p => p.theme.subText};
+  }
+`;
+
+const QueryWrapper = styled('div')`
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  width: 100%;
+  overflow: hidden;
+`;
+
+const Suggestion = styled('li')`
+  position: relative;
+  display: inline-grid;
+  grid-template-columns: 170px auto;
+  align-items: center;
+  padding: ${space(1)} 0;
+  border-bottom: 1px solid ${p => p.theme.innerBorder};
+  gap: ${space(1)};
+
+  &:hover {
+    cursor: pointer;
+  }
+
+  &:hover ${ActionsWrapper} {
+    visibility: visible;
+  }
+`;
+
+const SuggestionList = styled('ul')`
+  display: flex;
+  flex-direction: column;
+  padding: 0;
+`;
```

Comment:
once you've taken the title out of the list, it should make more sense to bring the grid up to the parent instead of applying an inline grid to each element and a margin left on StyledOverflowEllipsisTextContainer